rules/docker-telemetry.mk

```diff
@@ -13,3 +13,10 @@ $(DOCKER_TELEMETRY)_CONTAINER_NAME = telemetry
 $(DOCKER_TELEMETRY)_RUN_OPT += --net=host --privileged -t
 $(DOCKER_TELEMETRY)_RUN_OPT += -v /etc/sonic:/etc/sonic:ro
 
+$(DOCKER_TELEMETRY)_RUN_OPT += -v /var/log/wtmp:/var/log/wtmp:ro
+
+# enable docker cmd inside the telemetry container
+$(DOCKER_TELEMETRY)_RUN_OPT += -v /var/run/docker.sock:/var/run/docker.sock:rw
+$(DOCKER_TELEMETRY)_RUN_OPT += -v /usr/bin/docker:/usr/bin/docker
+$(DOCKER_TELEMETRY)_RUN_OPT += -v /usr/lib/x86_64-linux-gnu/libapparmor.so.1:/usr/lib/x86_64-linux-gnu/libapparmor.so.1
```

Comment:
why do we need this? can we install them in the docker instead of mapping them from the base image? also, what is docker cmd?

Reply:
We need to use docker command like "docker ps" to get docker container list, or "docker exec -i vtysh -c 'show ip bgp summary'" to get bgp neighbor status.

If we want to get the docker information or access the other docker container in the host, we should mapping them from host.

Reply:
letting an application in one docker to directly access cmd in another docker poses security concern. It is better to community this thought the database, in this case, we can ask bgp docker to put information into database and then telemetry can read from database.

Reply:
If you really need vtysh in a non-quaggy container, please let me know. We have some solution with socket.